newsfragments/3865.incompat

```diff
@@ -0,0 +1 @@
+Python 3.6 is no longer supported, as it has reached end-of-life and is no longer receiving security updates.
```

Comment:
Tahoe also has an "installation" type.  I see the reasoning behind using "incompat" here ... maybe we should have clearer (... ie, any) developer-facing docs about the intent for each of these fragment types.

Right now the next release has no installation-type fragments and only this incompat-type fragment so I suppose it's not a big deal which one we choose.

Reply:
If someone has an existing install, they will stop getting updates. So... it's not _just_ installation? I'll leave as is I guess.